dbus-glib: 0.108 -> 0.110

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools.

This update was made based on information from https://repology.org/metapackage/dbus-glib/versions.

These checks were done:

- built on NixOS
- Warning: no binary found that responded to help or version flags. (This warning appears even if the package isn't expected to have binaries.)
- found 0.110 with grep in /nix/store/745gjl73bz311i35h57d3slqd320bvjr-dbus-glib-0.110
- directory tree listing: https://gist.github.com/87d5b9565bc23aa3acfd68aaa8d88725

Commit:
dbus-glib: 0.108 -> 0.110

Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools.

This update was made based on information from https://repology.org/metapackage/dbus-glib/versions.

These checks were done:

- built on NixOS
- Warning: no binary found that responded to help or version flags. (This warning appears even if the package isn't expected to have binaries.)
- found 0.110 with grep in /nix/store/745gjl73bz311i35h57d3slqd320bvjr-dbus-glib-0.110
- directory tree listing: https://gist.github.com/87d5b9565bc23aa3acfd68aaa8d88725

FRidh commented:
This was reverted earlier because it did not build.
cc @Mic92

GrahamcOfBorg commented:
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38736&attempt_id=74cb0ae6-e749-4946-ab71-b6b67e42abeb)

Attempted: dbus-glib

<details><summary>Partial log (click to expand)</summary><p>

```
============================================================================
make[3]: *** [Makefile:833: test-suite.log] Error 1
make[3]: Leaving directory '/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make[2]: *** [Makefile:941: check-TESTS] Error 2
make[2]: Leaving directory '/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make[1]: *** [Makefile:1011: check-am] Error 2
make[1]: Leaving directory '/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make: *** [Makefile:524: check-recursive] Error 1
builder for ‘/nix/store/h56jrpnhld3ha6hcd4av2ah97nd03iyb-dbus-glib-0.110.drv’ failed with exit code 2
error: build of ‘/nix/store/h56jrpnhld3ha6hcd4av2ah97nd03iyb-dbus-glib-0.110.drv’ failed
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38736&attempt_id=0a924534-58b2-4f74-b74d-66ce6b374328)

Attempted: dbus-glib

<details><summary>Partial log (click to expand)</summary><p>

```
============================================================================
make[3]: *** [Makefile:833: test-suite.log] Error 1
make[3]: Leaving directory '/private/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make[2]: *** [Makefile:941: check-TESTS] Error 2
make[2]: Leaving directory '/private/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make[1]: *** [Makefile:1011: check-am] Error 2
make[1]: Leaving directory '/private/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make: *** [Makefile:524: check-recursive] Error 1
builder for '/nix/store/8xlfdlwiabhq11l0y27iwkgc7m100d96-dbus-glib-0.110.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/8xlfdlwiabhq11l0y27iwkgc7m100d96-dbus-glib-0.110.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38736&attempt_id=92713d19-96ee-4faa-b5dd-8feb01febfb4)

Attempted: dbus-glib

<details><summary>Partial log (click to expand)</summary><p>

```
============================================================================
make[3]: *** [Makefile:833: test-suite.log] Error 1
make[3]: Leaving directory '/build/dbus-glib-0.110/dbus-gmain'
make[2]: *** [Makefile:941: check-TESTS] Error 2
make[2]: Leaving directory '/build/dbus-glib-0.110/dbus-gmain'
make[1]: *** [Makefile:1011: check-am] Error 2
make[1]: Leaving directory '/build/dbus-glib-0.110/dbus-gmain'
make: *** [Makefile:524: check-recursive] Error 1
builder for '/nix/store/psjc7p2k3q6dmc3ffphig7cqdckgypv6-dbus-glib-0.110.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/psjc7p2k3q6dmc3ffphig7cqdckgypv6-dbus-glib-0.110.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38736&attempt_id=3e97a65c-11e8-4873-8f8a-d00592bbbfd3)

Attempted: dbus-glib

<details><summary>Partial log (click to expand)</summary><p>

```
============================================================================
make[3]: *** [Makefile:833: test-suite.log] Error 1
make[3]: Leaving directory '/build/dbus-glib-0.110/dbus-gmain'
make[2]: *** [Makefile:941: check-TESTS] Error 2
make[2]: Leaving directory '/build/dbus-glib-0.110/dbus-gmain'
make[1]: *** [Makefile:1011: check-am] Error 2
make[1]: Leaving directory '/build/dbus-glib-0.110/dbus-gmain'
make: *** [Makefile:524: check-recursive] Error 1
builder for '/nix/store/v53702smswr9l8rbj2m8rzj9iv2r8riq-dbus-glib-0.110.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/v53702smswr9l8rbj2m8rzj9iv2r8riq-dbus-glib-0.110.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38736&attempt_id=2b9e90d0-0776-47eb-90e6-0c66f142f232)

Attempted: dbus-glib

<details><summary>Partial log (click to expand)</summary><p>

```
============================================================================
make[3]: *** [Makefile:833: test-suite.log] Error 1
make[3]: Leaving directory '/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make[2]: *** [Makefile:941: check-TESTS] Error 2
make[2]: Leaving directory '/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make[1]: *** [Makefile:1011: check-am] Error 2
make[1]: Leaving directory '/tmp/nix-build-dbus-glib-0.110.drv-0/dbus-glib-0.110/dbus-gmain'
make: *** [Makefile:524: check-recursive] Error 1
builder for ‘/nix/store/kkjk9iwzyg6c19yf9p22szdblfrv4zvl-dbus-glib-0.110.drv’ failed with exit code 2
error: build of ‘/nix/store/kkjk9iwzyg6c19yf9p22szdblfrv4zvl-dbus-glib-0.110.drv’ failed
```
</p></details>

jtojnar commented:
NEWS

```
dbus-glib is deprecated, please use GDBus in new GLib-based projects.

dbus-glib 0.110 (2018-01-29)
============================

The “sphere tank” release.

Dependencies:

• GLib 2.40 is required

Enhancements:

• The GLib main-loop glue, "dbus-gmain", is now available as a separate
  subproject (the dbus-gmain branch in dbus-glib's git repository) for
  embedding in larger projects like dbus-glib and dbus-python via the
  `git subtree` or `git submodule` commands. This removes dbus-python's
  dependency on the rest of dbus-glib.

Fixes:

• autogen.sh can now detect gtk-doc >= 1.26.

• More files have per-file copyright information.
```

xeji commented:
@jtojnar Since it's deprecated anyway, why bother with an update? I think this can be closed.

jtojnar commented:
It is still widely used. And we will need it when we update gtk-doc.

xeji commented:
Well then let's fix it. Looks like it compiles but a test fails due to some path impurity. `checkPhase` runs fine locally in nix-shell. Do you think the test is worth fixing, or should we just `doCheck=false` ?

jtojnar commented:
`doCheck=false` should be fine.

xeji commented:
ok, I'll add that here

xeji committed:
dbus-glib: diable tests

Test fails in sandbox due to some path impurity but passes locally.
Not worth fixing since dbus-glib is deprecated.

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38736&attempt_id=46156c03-7e97-4f0a-a56f-b71453872a14)

Attempted: dbus-glib

<details><summary>Partial log (click to expand)</summary><p>

```
cannot build derivation '/nix/store/5fy1iwz9kbi3s0qybzvf7wbljhf5jn3v-python-2.7.15.drv': 15 dependencies couldn't be built
cannot build derivation '/nix/store/bz94f2vdl32w045s1fp590ilv5753z21-glib-2.56.0.drv': 11 dependencies couldn't be built
cannot build derivation '/nix/store/2grm6mmjlj5x547jq0xlp2lw124rkg2a-libxml2-2.9.8.drv': 5 dependencies couldn't be built
cannot build derivation '/nix/store/nkx7qv9j1aarzm3pajx1njyj1fp5j568-xcb-proto-1.12.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/cdw2bafrvfv17vhcvnxh5jjrgwh35x0x-libxslt-1.1.32.drv': 5 dependencies couldn't be built
cannot build derivation '/nix/store/g26fqxvkdsh9ay105xbdffh7bznqxf0l-libxcb-1.12.drv': 9 dependencies couldn't be built
cannot build derivation '/nix/store/j1x2ffgamsn0frypqgwiq70rvyb09ykj-libX11-1.6.5.drv': 10 dependencies couldn't be built
cannot build derivation '/nix/store/3r8ydv76yc9dqgmdxg9z1m8yk3dr7agv-dbus-1.12.8.drv': 7 dependencies couldn't be built
cannot build derivation '/nix/store/p9zrmdqa53s4ki6v3dhf25hwcdng82p7-dbus-glib-0.110.drv': 8 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/p9zrmdqa53s4ki6v3dhf25hwcdng82p7-dbus-glib-0.110.drv' failed
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38736&attempt_id=be5beca0-7502-44e7-91af-31abaec4ce2d)

Attempted: dbus-glib

<details><summary>Partial log (click to expand)</summary><p>

```
shrinking /nix/store/jv3n5p432dvlcsjxma7jh62b5a6yvkph-dbus-glib-0.110-dev/bin/dbus-binding-tool
gzipping man pages under /nix/store/jv3n5p432dvlcsjxma7jh62b5a6yvkph-dbus-glib-0.110-dev/share/man/
strip is /nix/store/w5sr51ad8kkm1smf8pxz5nfca271vzkw-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/jv3n5p432dvlcsjxma7jh62b5a6yvkph-dbus-glib-0.110-dev/lib  /nix/store/jv3n5p432dvlcsjxma7jh62b5a6yvkph-dbus-glib-0.110-dev/bin
patching script interpreter paths in /nix/store/jv3n5p432dvlcsjxma7jh62b5a6yvkph-dbus-glib-0.110-dev
checking for references to /build in /nix/store/jv3n5p432dvlcsjxma7jh62b5a6yvkph-dbus-glib-0.110-dev...
shrinking RPATHs of ELF executables and libraries in /nix/store/5i0jlbrz53cm6fqd0j6qwmbi2cj1vlr7-dbus-glib-0.110-devdoc
strip is /nix/store/w5sr51ad8kkm1smf8pxz5nfca271vzkw-binutils-2.30/bin/strip
patching script interpreter paths in /nix/store/5i0jlbrz53cm6fqd0j6qwmbi2cj1vlr7-dbus-glib-0.110-devdoc
checking for references to /build in /nix/store/5i0jlbrz53cm6fqd0j6qwmbi2cj1vlr7-dbus-glib-0.110-devdoc...
```
</p></details>

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38736&attempt_id=b0595e16-fb5d-478e-bfec-aa3f9111e289)

Attempted: dbus-glib

<details><summary>Partial log (click to expand)</summary><p>

```
gzipping man pages under /nix/store/wask1vlpxnjsk717q0bkccn38xnrl4x7-dbus-glib-0.110-dev/share/man/
strip is /nix/store/6acxxjiawywkry256piddiskp13j1kl2-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/wask1vlpxnjsk717q0bkccn38xnrl4x7-dbus-glib-0.110-dev/lib  /nix/store/wask1vlpxnjsk717q0bkccn38xnrl4x7-dbus-glib-0.110-dev/bin
patching script interpreter paths in /nix/store/wask1vlpxnjsk717q0bkccn38xnrl4x7-dbus-glib-0.110-dev
checking for references to /build in /nix/store/wask1vlpxnjsk717q0bkccn38xnrl4x7-dbus-glib-0.110-dev...
shrinking RPATHs of ELF executables and libraries in /nix/store/9qzvv7jspgbzlqws0yw0z944m1glindz-dbus-glib-0.110-devdoc
strip is /nix/store/6acxxjiawywkry256piddiskp13j1kl2-binutils-2.30/bin/strip
patching script interpreter paths in /nix/store/9qzvv7jspgbzlqws0yw0z944m1glindz-dbus-glib-0.110-devdoc
checking for references to /build in /nix/store/9qzvv7jspgbzlqws0yw0z944m1glindz-dbus-glib-0.110-devdoc...
/nix/store/1wlgw15bqf0a914nh5gd7fvklgcwr30p-dbus-glib-0.110
```
</p></details>

xeji commented:
Looks like lots of Darwin things are  broken in staging...
